communityCustomizations/ShowPasswordButton/OnLoad.js

```diff
@@ -0,0 +1,15 @@
+var inputArea = document.getElementById("inputArea");
+
+if(inputArea){
+     console.log("hello");
+     var showButton = document.createElement("div");
+ showButton.id = "showButton";  
```

Comment:
Would you mind adjusting the indent here to match for the if statement

Reply:
Thank you, I'll do